user-guides/ivche1337/Setting-up-your-organization

```diff
@@ -0,0 +1,9 @@
+Setting up your organization
+
+All the topics needed in order to setup a healthy zulip organisations are listed.
```

Comment:
Capitalize Zulip, and spell `organization` with a `z` (Zulip uses American English).